Add watchface analog-brawnish

David-Rinehart opened the pull request:
Add new watchface for analog-brawnish

David-Rinehart committed:
Add analog-brawnish

eLtMosen commented on analog-brawnish/usr/share/asteroid-launcher/watchfaces/analog-brawnish.qml:
```diff
@@ -0,0 +1,259 @@
+/*
+ * Copyright (C) 2022 - David Rinehart wdaver(at)cox.net
```
The RotM and CenterX/Y variables later on sound rather familiar. If you built this watchface using an example with given credits, it would be nice to the previous contributors to keep those mentions according to the license and extend your credits on top.
Don't take this as an offense please, if you just used the variable names but consider the code your sole work, there is nothing to complain ;)

moWerk reviewed (changes requested):
Like you mentioned on matrix, the README.md is missing your entry.
Thank you very much for your contribution!

moWerk committed:
Fix slogan, scaling, round screen clip and DRY refactor

Correct slogan from Hack Your Wrist to Free Your Wrist.

Replace hardcoded 800x800 background dimensions with anchors.fill
so the face scales correctly at any resolution. Add DeviceSpecs
round screen radius clip to background rectangle. Add background
opacity 0.5. Import org.asteroid.utils 1.0 for DeviceSpecs access.

Lift hours, minutes and seconds to root-level properties and remove
the redundant per-delegate property var declarations throughout.

Add id to all top-level elements. Normalize mixed tab and space
indentation to spaces. Uppercase hex color literals for consistency.

moWerk committed:
Remove unused import, tighten types and scale with Dims.l

Remove unused QtGraphicalEffects import. Change root time properties
from property var to property int. Replace width and height if
expressions in ticks delegate with ternary operators.

Add org.asteroid.controls import for Dims access. Set root Item
width and height to Dims.l(100) so all parent.width and parent.height
fractions resolve correctly across different screen sizes. Convert
hand and shadow radius values from hardcoded pixels to Dims.l assuming
a 360px design base. Convert shadow centerX and centerY pixel offsets
to Dims.l equivalents.

moWerk reviewed (approved):
Resolved the issues after this PR sat idle for very long.

moWerk committed:
Add analog-brawnish to README watchface table and font license row, clean up Michroma row and add thumbnails for analog-brawnish
